rpcserver.go

```diff
@@ -1693,7 +1693,7 @@ func (r *rpcServer) SendPayment(paymentStream lnrpc.Lightning_SendPaymentServer)
 				// attempt to decode it, populating the
 				// payment accordingly.
 				if nextPayment.PaymentRequest != "" {
-					payReq, err := zpay32.Decode(nextPayment.PaymentRequest)
+					payReq, err := zpay32.Decode(nextPayment.PaymentRequest, activeNetParams.Params)
```

Comment:
line too long (should be 80 chars max, with 8 width tabs)

Reply:
I'll keep this in mind going forward, I thought `gofmt` would catch this but evidently not.